x-pack/plugins/rollup/server/plugin.ts

```diff
@@ -100,11 +103,14 @@ export class RollupPlugin implements Plugin<void, void, any, any> {
       }
     }
 
-    if (indexManagement && indexManagement.indexDataEnricher) {
-      indexManagement.indexDataEnricher.add(rollupDataEnricher);
+    if (this.config.ui.enabled) {
+      if (indexManagement && indexManagement.indexDataEnricher) {
+        indexManagement.indexDataEnricher.add(rollupDataEnricher);
+      }
+
+      dataViews.enableRollups();
+      data.search.enableRollups();
```

Comment:
Hey @mattkime, just wanted to confirm with you that it's correct to not enable Rollups in data views and data search, if the config `xpack.rollup.ui.enabled` is set to `false`. The same is already done on the client side.

Reply:
Yes, I think this is the right thing to do but we should be aware that this is a change. Previously, only the UI was recognizing the `xpack.rollup.ui.enabled` value, the server ignored it.

Reply:
Thanks for the feedback, @mattkime! Yes, that is definitely a change, I've added it to the PR description for clarity.

Reply:
> Yes, I think this is the right thing to do but we should be aware that this is a change. Previously, only the UI was recognizing the xpack.rollup.ui.enabled value, the server ignored it.

That's a good point, actually. Would this be considered a breaking change? 🤔 I'm trying to see if this config is publicly documented, but I don't see anything.

Reply:
>Would this be considered a breaking change?

The only way I could see this breaking something is if something was making rollup related requests to the kibana server WHILE THE ROLLUP UI WAS DISABLED. 

Things surprise me every day but I think this is unlikely enough that we can assume it won't happen.

Reply:
Yeah, that makes sense. Just wanted to double check 👍